include/mainwindow.h

```diff
@@ -1,11 +1,14 @@
 #ifndef MAINWINDOW_H
 #define MAINWINDOW_H
 
-#define USE_TEST_DATA_1
-// #define USE_TEST_DATA_2  // Using Test Data 1
+// #define USE_TEST_DATA_1
+// #define USE_TEST_DATA_2
```

Comment:
It seems a little odd to have both of these commented out. Should they just be removed at this point?

Reply:
The #define directive and its associated code have been purged.